lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -125,7 +125,7 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
         return None
 
     patient_entry, patient_reference = create_patient(enrollment)
-    birthdate = parse_birth_date(enrollment)
+    birthdate = parse_date_from_string(enrollment.get('core_birthdate'))
```

Comment:
Since the log level was downgraded from warning > debug inside the method to make it more general purpose, should we check here that birthdate is not null and otherwise warn? I wonder if having a birthdate we couldn't parse is a case we would need to take action on to manually debug/fix.

Reply:
That's a really good idea. We would want to investigate an HCT record that didn't have its `core_birthdate`. I added a LOG.warning and am continuing to process the record.